lib/src/ui/meeting/hms_video_view.dart

```diff
@@ -18,9 +18,14 @@ class HMSVideoView extends StatelessWidget {
   /// [HMSVideoView] will use viewSize to get height and width of rendered video. If not passed, it will take whatever size is available to the widget.
   final Size? viewSize;
   bool setMirror;
+  final bool isScreenShare;
```

Comment:
this info can be taken from the HMSTrack's source property

Reply:
resolved

Reply:
hms_video_scale_type is missing

Reply:
resolved